src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,275 @@
+
+<br />
+
+<CardGrid gridNumber={3} mobileGridNumber={1} colsMinWidth='150px' colsMinWidthMobile='100px'>
+    <TitleCard icon={githubLogo} title="GitHub" link="/configuration/git-providers#github" imageAlt="'GitHub logo"/>
+    <TitleCard icon={gitlabLogo} title="GitLab" link="/configuration/git-providers#gitlab" imageAlt="GitLab logo"/>
+    <TitleCard icon={bitbucketLogo} title="Bitbucket" link="/configuration/git-providers#bitbucket" imageAlt="BitBucket logo"/>
+</CardGrid>
+
+For more details on available Git Providers and related commands, use the `daytona git-providers` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Add a Git Provider"
+    subtitle="Learn how to add a Git Provider to Daytona and start managing your repositories effectively."
+    href="/configuration/git-providers"
+    expanded
+  />
+</DocumentList>
+
+## Creating your Workspace
+
+Once you have completed the initial setup and configuration steps, you can [create your Workspace](/usage/workspaces#create-a-workspace) using Daytona. A Workspace represents an isolated development environment where you can manage your code, build projects, and collaborate with team members.
+
+Creating a Workspace involves defining the necessary configurations, such as the Git repository URL and selecting the appropriate [Git Provider](/configuration/git-providers). Daytona simplifies this process by providing a user-friendly interface to set up and manage your Workspaces efficiently.
+
+To create a Workspace, use the `daytona create` command and follow the prompts to configure your Workspace settings. Once created, you can access your Workspace and interact with your codebase.
+
+<DocumentList>
+<DocumentListItem
+    title="Create a Workspace"
+    subtitle="Learn how to create your Workspace using Daytona."
+    href="/usage/workspaces"
+    expanded
+  />
+</DocumentList>
+
+## Installing a Provider
+
+A [Provider](/configuration/providers) is a plugin that interfaces with the [Daytona Server](/usage/server), handling development environment deployment and lifecycle management.
+
+Daytona supports a variety of Providers, allowing you to integrate with different platforms based on your needs. Once installed, a Provider will manage the deployment of your development environment directly within Daytona, ensuring integration with your chosen platform.
+
+For more details on available Providers and related commands, use the `daytona provider` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Install a Provider"
+    subtitle="Learn how to install a Provider and interface with the Daytona Server."
+    href="/configuration/git-providers"
```

Comment:
@stefanicjuraj this link goes to GIT Providers and not Providers. 

Dropping a note to @Tpuljak as once again the GIT Providers / Providers naming can be confusing